pkg/bundles/kontext/bundle.go

```diff
@@ -50,10 +52,23 @@ func bundle(directory string) (v1.Layer, error) {
 				return err
 			}
 
-			// Skip anything in the .git directory
-			// TODO(mattmoor): expand this to .gitignore / .dockerignore?
-			if fi.IsDir() && filepath.Base(path) == ".git" {
+			ignoreScanner, err := ignore.NewOrDefault(directory)
```

Comment:
Can we call this `ignorer`?

Reply:
@mattmoor  - it will be great if you review the logic of the regex, I used some bases from docker code base for it , but I thought to ask for a review